project-base/storefront/pages/products/[productSlug].tsx

```diff
@@ -46,6 +47,7 @@ const ProductDetailPage: NextPage = () => {
             {product?.__typename === 'MainVariant' && (
                 <ProductDetailMainVariantContent fetching={fetching} product={product} />
             )}
+            <LastVisitedProducts removeCatalogNumber={product?.catalogNumber} />
```

Comment:
Why don't we handle also main variant? 🤔

Reply:
I dont understand this part. It is part of Common Layout on bottom section. MainVariant content and Regular product content is above. I dont think this is restricted for any type of product.

Reply:
Maybe I got it. Last visited setter moved to productSlug, so it will set cookies for all product types.